workers/grouper/src/index.ts

```diff
@@ -54,23 +45,6 @@ export default class GrouperWorker extends Worker {
    */
   private redis = new RedisHelper();
 
-  /**
-   * Get unique hash from event data
-   *
-   * @param task - worker task to create hash
-   */
-  private static getUniqueEventHash(task: GroupWorkerTask): string {
-    const computedHashValueCacheKey = `${task.catcherType}:${task.event.title}`;
-
-    if (!this.cachedHashValues[computedHashValueCacheKey]) {
-      this.cachedHashValues[computedHashValueCacheKey] = crypto.createHmac('sha256', process.env.EVENT_SECRET)
```

Comment:
there was a memory leak, because `this.cachedHashValues` never flushed. I've replaced it with `this.cache` module